backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/objcexport/ObjCExportMapper.kt

```diff
@@ -316,12 +316,23 @@ private fun ObjCExportMapper.bridgeMethodImpl(descriptor: FunctionDescriptor): M
 
     val returnBridge = bridgeReturnType(descriptor, convertExceptionsToErrors)
     if (convertExceptionsToErrors) {
-        valueParameters += MethodBridgeValueParameter.ErrorOutParameter
+        // Add error out parameter before tail block parameters. The convention allows this.
+        // Placing it after would trigger https://github.com/JetBrains/kotlin-native/issues/3825
```

Comment:
Do we know Swift bug number here?

Reply:
I've reported it and improved the comment.